### PR TITLE
Incorporate the canMakePayment algorithm of the payment method good practices doc.

### LINKED ISSUE #309: Need normative text for checking the origin of the SW scope against "supported_origins".
From [TAG review](https://github.com/w3ctag/design-reviews/issues/231#issuecomment-377645336):

> [8.5 Supported Origin](https://w3c.github.io/payment-handler/#supported-origin) should link to the normative text that it refers to.

Comparing the origin of a service worker registration scope to the `"supported_origins"` list in a payment method manifest sounds like an algorithm. It should be defined in a normative section and references from the security section ([8.5](https://w3c.github.io/payment-handler/#supported-origin)).

cc @dbaron

### PR DESCRIPTION
**Pull request by rsolomakhin**
Closes #309.

The following tasks have been completed:

 * [ ] web platform tests (link)
 * [ ] MDN Docs added (link)

Implementation commitment:

 * [ ] Safari (link to issue)
 * [x] [Chrome](https://cs.chromium.org/chromium/src/components/payments/content/manifest_verifier.h?rcl=8cdb7d659c0d6b08352ee471f2c441de8122c9b6)
 * [ ] Firefox (link to issue)
 * [ ] Edge (public signal)

***
<a href="https://pr-preview.s3.amazonaws.com/rsolomakhin/payment-handler/pull/313.html" title="Last updated on Aug 17, 2018, 2:44 PM GMT (f7ce8f4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-handler/313/ce4e620...rsolomakhin:f7ce8f4.html" title="Last updated on Aug 17, 2018, 2:44 PM GMT (f7ce8f4)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1047,17 +1047,20 @@ <h2>
               </li>
             </ol>
           </li>
-          <li>Otherwise, fire the <a>CanMakePaymentEvent</a> in the payment
-          handler and return the result.
+          <li>Otherwise, if the <a>URL-based payment method identifier</a> and
+          payment handler share the same <a data-cite=
```
Comment (ianbjacobs) [inline index.html]: Is "payment handler" enough, or do you want to say something like "scope URL of the payment handler"?

File: index.html [hunk 2]
```diff
@@ -1047,17 +1047,20 @@ <h2>
...
+          "!HTML#origin">origin</a>, fire the <a>CanMakePaymentEvent</a> in the
+          payment handler and return the result.
+          </li>
+          <li>Otherwise, if <code>"supported_origins"</code> value in the
+          payment method manifest includes the <a data-cite=
+          "HTML#origin">origin</a> of the payment handler, fire the
```
Comment (ianbjacobs) [inline index.html]: Same observation here: "of the scope URL of the payment handler"?

File: index.html [hunk 3]
```diff
@@ -2029,7 +2032,7 @@ <h2>
         </h2>
         <ul>
           <li>The party responsible for a payment method authorizes payment
-          apps through a [[payment-method-manifest]]. See the <a>Handling a
+          apps through a payment-method-manifest. See the <a>Handling a
```
Comment (ianbjacobs) [inline index.html]: Was this edit intentional? I think a reference to that spec here makes sense.

**Review by ianbjacobs — COMMENTED** [review]
Thank you, Rouslan. Some minor comments inline.

File: index.html (hunk 1 above)
Comment (rsolomakhin) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (rsolomakhin) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (rsolomakhin) [reply index.html]: Done.

**Comment by rsolomakhin** [comment]
@ianbjacobs Please take another look. I've addressed your comments.

**Review by ianbjacobs — APPROVED** [review]

**Comment by rsolomakhin** [comment]
@romandev Please take a look. Is this OK to merge?

**Review by romandev — APPROVED** [review]
lgtm!
